### PR TITLE
Fix/stack-name-added_continued

### PR DESCRIPTION
**Pull request by MariaAngella**
Hello,

I am trying to resolve the merge conflicts as per issue [194](https://github.com/odk-x/tool-suite-X/issues/194).
Steps done;
1. Forked repo and checked out from development branch 
2. Edited 2 files init-odkx-sync-endpoint.py and https.env with changes made by @adityaofficial10 

Please guide if I'm on the right track.

Thank you and regards,
Maria Angella

### CODE REVIEW

**Comment by wbrunette** [comment]
The env variable is array that has all the variables in it.

**Comment by MariaAngella** [comment]
@wbrunette kindly review changes made.

Thanks and regards,
Maria Angella

File: init-odkx-sync-endpoint.py [hunk 1]
```diff
@@ -14,11 +14,11 @@
 from os import fdopen, remove, path
 from xml import dom
 
-def run_interactive_config():
+def run_interactive_config(env):
     env_file_location = os.path.join(os.path.dirname(__file__), "config", "https.env")
 
     try:
-        env = parse_env_file(env_file_location)
+        env, domain, email = parse_env_file(env_file_location)
```
Comment (wbrunette) [inline init-odkx-sync-endpoint.py:21]: this call should only return on variable, so i am confused on why 3 variables.

**Review by wbrunette — CHANGES_REQUESTED** [review]
I am not sure this will work. You should test your code. 

To just have the code be the text parsing part. I suggest.
If you comment out  these functions in main, the code will only run the interactive questioning. 
run_docker_builds()
    run_sync_endpoint_build()
    deploy_stack(https)

The other part is the "if enforce_https:" if you set that to false the script will not try to run code to get the certificate

**Comment by MariaAngella** [comment]
@wbrunette Thank you for the advice and patience.

Work still on progress Please hope to resolve this issue before 30th Oct 2023.

**Pull request closed without merging**